File: app/src/main/java/capstone/design/message/MessageType.java
package capstone.design.message;

public enum MessageType {
    REQ_PULL, RES_PULL, // topic/partition에서 메시지 요청
    REQ_PUSH, RES_PUSH, // topic/partition에 메시지 저장
    REQ_FIND, RES_FIND, // topic/partition에서 메시지 검색
    REQ_SEEK, RES_SEEK; // topic/partition에서 특정 오프셋으로 이동

    public byte getByte() { return (byte) this.ordinal(); }
    public static int SIZE = Byte.BYTES;
}

File: services/convert_service/py_client/message/message_type.py
from enum import IntEnum

class MessageType(IntEnum):
    REQ_PULL        = 0
    RES_PULL        = 1

    REQ_PUSH        = 2
    RES_PUSH        = 3

    REQ_FIND        = 4
    RES_FIND        = 5

    REQ_SEEK        = 6
    RES_SEEK        = 7